query-coordinator/src/test/scala/com/socrata/querycoordinator/TestQueryRewriter.scala

```diff
@@ -534,4 +535,10 @@ class TestQueryRewriter extends TestBase with TestCompoundQueryRewriterBase {
     val rewrittenQuery = "SELECT c1 as number1, c2 as sum_number1 LIMIT 2"
     checkQueryRewrite(q, rollups, "ru1", rewrittenQuery)
   }
+
+  test("rewrite with piped rollup query") {
+    val q = "SELECT ward, crime_type, number1 WHERE number1='0' |> SELECT ward, crime_type WHERE crime_type='traffic'"
```

Comment:
Q: why does the WHERE condition get dropped altogether from the rewritten query?